lib/ts/recipe/session/index.ts

```diff
@@ -257,31 +264,38 @@ export default class SessionWrapper {
         accessToken: string,
         antiCsrfToken?: string,
         options?: VerifySessionOptions & { sessionRequired?: true },
+        antiCSRF?: AntiCsrfType,
```

Comment:
This needs better naming, it can easily be confused with antiCsrfToken (This should be fixed everywhere in this file)